services/go-importer/cmd/assembler/main.go

```diff
@@ -34,8 +35,16 @@ var promisc = true
 var watch_dir = flag.String("dir", "", "Directory to watch for new pcaps")
 var mongodb = flag.String("mongo", "", "MongoDB dns name + port (e.g. mongo:27017)")
 var flag_regex = flag.String("flag", "", "flag regex, used for flag in/out tagging")
+var pcap_over_ip = flag.String("pcap-over-ip", "", "PCAP-over-IP host + port (e.g. remote:1337)")
+var bpf = flag.String("bpf", "", "BPF filter")
 var nonstrict = flag.Bool("nonstrict", false, "Do not check strict TCP / FSM flags")
 var experimental = flag.Bool("experimental", false, "Enable experimental features.")
+var flushAfter = flag.String("flush-after", "", `
+Connections which have buffered packets (they've gotten packets out of order and
+are waiting for old packets to fill the gaps) can be flushed after they're this old
+(their oldest gap is skipped). This is particularly useful for pcap-over-ip captures.
+Any string parsed by time. ParseDuration is acceptable here. No flushing is done if
```

Comment:
```suggestion
Any string parsed by time.ParseDuration is acceptable here (ie. "3m", "2h45m"). No flushing is done if
```
Examples might be nice for infra guys that don't know golang.